doc/api/path.md

```diff
@@ -2,31 +2,85 @@
 
     Stability: 2 - Stable
 
-This module contains utilities for handling and transforming file
-paths. The file system is not consulted to check whether paths are valid.
+The `path` module provides utilities for working with file paths. It can be
+accessed using:
 
-Use `require('path')` to use this module.  The following methods are provided:
+```js
+const path = require('path');
+```
+
+## Windows vs. POSIX
+
+The default operation of the `path` module varies based on the operating system
+on which a Node.js application is running. Specifically, when running on a
+Windows operating system, the `path` module will assume that Window-style
+paths are being used.
+
+For example, using the `path.basename()` function with the Windows file path
+`C:\temp\myfile.html`, will yield different results when running on POSIX than
+when run on Windows:
+
+On POSIX:
+
+```js
+path.basename('C:\\temp\\myfile.html');
+  // returns 'C:\temp\myfile.html'
+```
+
+On Windows:
+
+```js
+path.basename('C:\\temp\\myfile.html');
+  // returns 'myfile.html'
+```
+
+To achieve consistent results when working with Windows file paths on any
+operating system, use [`path.win32`][]:
+
+On POSIX and Windows:
+```js
+path.win32.basename('C:\\temp\\myfile.html');
+  // returns 'myfile.html'
+```
+
+To achieve consistent results when working with POSIX file paths on any
+operating system, use [`path.posix`]:
+
+On POSIX and Windows:
+```js
+path.posix.basename('/tmp/myfile.html');
+  // returns 'myfile.html'
+```
 
 ## path.basename(path[, ext])
 
-Return the last portion of a path, similar to the Unix `basename` command.
-`path` must be a string. `ext`, if given, must also be a string.
+* `path` {String} A file path
+* `ext` {String} An optional file extension
 
-Examples:
+The `path.basename()` methods returns the last portion of a `path`, similar to
+the Unix `basename` command.
+
+For example:
 
 ```js
 path.basename('/foo/bar/baz/asdf/quux.html')
-// returns 'quux.html'
+  // returns 'quux.html'
 
 path.basename('/foo/bar/baz/asdf/quux.html', '.html')
-// returns 'quux'
+  // returns 'quux'
 ```
 
+A [`TypeError`][] is thrown if `path` is not a string or if `ext` is given
+and is not a string.
+
 ## path.delimiter
 
-The platform-specific path delimiter, `;` or `':'`.
+Returns the platform-specific path delimiter:
```

Comment:
Could the ``returns`` word raise confusion on the reader and let them think that this is a function?